src/main/java/edu/kis/powp/jobs2d/drivers/canva/WorkspaceCanva.java

```diff
@@ -0,0 +1,54 @@
+package edu.kis.powp.jobs2d.drivers.canva;
+
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.adapter.LineDriverAdapter;
+import edu.kis.powp.jobs2d.drivers.canva.shapes.CanvaShape;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
+
+
+public class WorkspaceCanva implements Job2dDriver {
+    private final Job2dDriver innerDriver;
+
+    private final Job2dDriver borderDriver;
+
+    private int X, Y;
+    private CanvaShape bound;
```

Comment:
Adding "Workspace feature" it is important to provide access to current canvas setting to the whole application so other features can take advantege of it.